core/ibft.go

```diff
@@ -747,15 +750,14 @@ func (i *IBFT) runPrepare(ctx context.Context) error {
 		// Grab the current view
 		view = i.state.getView()
 
-		// Grab quorum information
-		quorum = i.backend.Quorum(view.Height)
-
 		// Subscribe to PREPARE messages
 		sub = i.messages.Subscribe(
 			messages.SubscriptionDetails{
```

Comment:
do we need a constuctor?

Reply:
Agree, it would be better, but let's keeps this PR scope to the minimum and note this refactor.

Reply:
ok.